### PR TITLE
build: Add release and test actions

### LINKED ISSUE #2: Automate Testing and Release Process
We should automate our release process in the same way we publish other NPM packages. We can configure a github action in the same way as as [fastify-hmac](https://github.com/autotelic/fastify-hmac/blob/main/.github/workflows/release.yml)

We should also add the `test` github action outlined in that same repo.

### PR DESCRIPTION
**Pull request by megquirk**
### Summary 

- Resolves #2 
- Add  actions to automate testing and release processes

### CODE REVIEW

File: .github/workflows/release.yml [hunk 1]
```diff
@@ -0,0 +1,35 @@
...
+      matrix:
+        node-version: [12.x, 14.x]
+
+    steps:
+      - uses: actions/checkout@v2
+      - name: Use Node.js ${{ matrix.node-version }}
+        uses: actions/setup-node@v1
+        with:
+          node-version: ${{ matrix.node-version }}
+      - run: npm ci
```
Comment (jkirkpatrick24) [inline .github/workflows/release.yml]:
```suggestion
      - run: npm install
```

File: .github/workflows/test.yml [hunk 2]
```diff
@@ -0,0 +1,25 @@
...
+      matrix:
+        node-version: [12.x, 14.x]
+
+    steps:
+      - uses: actions/checkout@v2
+      - name: Use Node.js ${{ matrix.node-version }}
+        uses: actions/setup-node@v1
+        with:
+          node-version: ${{ matrix.node-version }}
+      - run: npm ci
```
Comment (jkirkpatrick24) [inline .github/workflows/test.yml]:
```suggestion
      - run: npm install
```

**Comment by megquirk** [comment]
changes implemented

File: .github/workflows/release.yml [hunk 3]
```diff
@@ -0,0 +1,35 @@
...
+
+    steps:
+      - uses: actions/checkout@v2
+      - name: Use Node.js ${{ matrix.node-version }}
+        uses: actions/setup-node@v1
+        with:
+          node-version: ${{ matrix.node-version }}
+      - run: npm install
+      - run: npm run lint
+      - run: npm test
```
Comment (jkirkpatrick24) [inline .github/workflows/release.yml:35]: Because we run the tests in this action it also will need a postgres service to connect to.

**Comment by megquirk** [comment]
Updated release to include postgres service

**Review by jkirkpatrick24 — APPROVED** [review]
LGTM

**Pull request merged**